recipes/dateutils/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "0.6.6" %}
+
+package:
+    name: dateutils
+    version: {{ version }}
+
+source:
+    fn: dateutils-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/d/dateutils/dateutils-{{ version }}.tar.gz
+    md5: 2ba7fcac03635f1f1cad0d94d785001b
+
+build:
+    number: 0
+    skip: True  # [py3k]
+    script: python setup.py install --single-version-externally-managed --record record.txt
+    entry_points:
+        - dateadd = dateutils.dateadd:main
+        - datediff = dateutils.datediff:main
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+        - argparse  # [py26]
+        - python-dateutil
+        - pytz
+
+test:
+    imports:
+        - dateutils
+    commands:
+        - dateadd --help
+        - datediff --help
+
+about:
+    home: https://pypi.python.org/pypi/dateutils/
+    license: BSD
```

Comment:
There is no description of what BSD type and the package was last uploaded in 2013!